admin/howto/support.md

```diff
@@ -1,52 +1,31 @@
-:text: Update/add packages, infrastructure, etc.
-:classes: stretched-link
-
----
-Report an issue 🐛
-^^^
-```{link-button} https://github.com/2i2c-org/pilot/issues/new?labels=bug&template=tech-support.md
-:text: Crashing sessions, broken webpages, etc.
-:classes: stretched-link
----
-Ask a question ❓
-^^^
-```{link-button} https://github.com/2i2c-org/pilot/issues/new?labels=question&template=questions.md
-:classes: stretched-link
-:text: Generic questions about the hub.
-```
-````
+If a problem persists after attempting to solve it, or if the hub requires a change that the Community Champion cannot implement themselves, they should reach out to the 2i2c support team.
+Hub engineers can provide support and debugging for issues that are related to the 2i2c Hub infrastructure.
+There are a few ways to get in touch listed below (in decreasing order of preference!)
 
-:::{note}
-If you have a support question that shouldn't be in public, please [send a support email](support:email).
-:::
+(support:issues)=
+## File a support ticket
 
-(support:slack)=
-## Connect on Slack
+We handle most support requests via [GitHub issues in the `support/` repository](https://github.com/2i2c-org/support/issues).
+This is a **private repository** that is accessible to Community Champions and 2i2c Engineers.
 
-You're also welcome to connect with the 2i2c Hub Engineering team via [the 2i2c Slack](https://2i2c.slack.com) (in particular, check out the `#support` channel). If you don't have access to the Slack already, [open an issue in the pilot repository](https://github.com/2i2c-org/pilot/issues/new) and we're happy to give you access.
+[**Click here to file a support ticket**](https://github.com/2i2c-org/support/issues/new) or go to [support.2i2c.org](http://support.2i2c.org).
 
+:::{note}
+If you have a support question with sensitive or private information involved, please [send a support email](support:email).
+:::
 
 (support:email)=
 ## Send a support email
 
-For support requests that should not be public, you can also [send the 2i2c engineering team an email via `support@2i2c.org`](mailto:support@2i2c.org).
-We prefer to handle support requests via public issue tracking, but understand that this is not always possible!
+For support requests that have sensitive information, you can also [send the 2i2c support team an email via `support@2i2c.org`](mailto:support@2i2c.org).
+We prefer to handle support requests via the `support/` repository tracking, but understand that this is not always possible!
+
+(support:slack)=
+## Connect on Slack
+
+You're also welcome to connect with the 2i2c Hub Engineering team via [the 2i2c Slack](https://2i2c.slack.com) (in particular, check out the `#support` channel). If you don't have access to the Slack already, [open an issue in the support repository](https://github.com/2i2c-org/support/issues/new) and we're happy to give you access.
```

Comment:
Maybe we can word this in another way to discourage dropping issues on Slack... IMHO, the Slack support channel should be a place to talk with the community champion to gather **additional** information...